src/chat/components/InputToolbar.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import {
+  View,
+  StyleSheet,
+  ScrollView,
+  StyleProp,
+  ViewStyle,
+  ImageStyle,
+} from 'react-native';
+import {
+  Composer,
+  InputToolbarProps,
+  SendProps,
+} from 'react-native-gifted-chat';
+import { PressableIcon } from './PressableIcon';
+
+const ImageURL = {
+  camera: require('../../images/camera.png'),
+  gallery: require('../../images/gallery.png'),
+  send: require('../../images/send.png'),
+};
+export interface IInputToolbar extends InputToolbarProps<any>, SendProps<any> {
+  isShowCamera?: boolean;
+  isShowPhotoGallery?: boolean;
+  togglePhotoGallery?: () => void;
+  containerStyle?: StyleProp<ViewStyle>;
+  composeWrapperStyle?: StyleProp<ViewStyle>;
+  composerTextInputStyle?: StyleProp<ViewStyle>;
+  customViewStyle?: StyleProp<ViewStyle>;
+  iconCamera?: any;
+  iconGallery?: any;
+  iconSend?: any;
+  iconStyle?: StyleProp<ImageStyle>;
+}
+
+const InputToolbar: React.FC<IInputToolbar> = ({
+  isShowCamera = true,
+  isShowPhotoGallery = true,
+  togglePhotoGallery,
+  containerStyle,
+  composeWrapperStyle,
+  composerTextInputStyle,
+  iconCamera = ImageURL.camera,
+  iconGallery = ImageURL.gallery,
+  iconSend = ImageURL.send,
+  iconStyle,
+  ...props
+}) => {
+  const { onSend, text } = props;
+
+  return (
+    <View style={[styles.container, containerStyle]}>
+      {isShowCamera && (
+        <PressableIcon
+          icon={iconCamera}
+          iconStyle={(styles.iconStyleDefault, iconStyle)}
+        />
+      )}
+      {isShowPhotoGallery && (
+        <PressableIcon
+          onPress={togglePhotoGallery}
+          icon={iconGallery}
+          iconStyle={[styles.iconStyleDefault, iconStyle]}
+        />
+      )}
+      <View style={[styles.composeWrapper, composeWrapperStyle]}>
+        <ScrollView scrollEnabled={false}>
+          <Composer
+            {...props}
+            textInputStyle={[styles.textInput, composerTextInputStyle]}
+          />
+        </ScrollView>
+      </View>
+      {!!text && (
+        <PressableIcon
+          iconStyle={[styles.iconStyleDefault, iconStyle]}
```

Comment:
I recommend use `StyleSheet.flatten([...])` for all style that we allow customize

Reply:
ok I updated it